README.md

```diff
@@ -307,6 +314,11 @@ Here is the outline of the configuration options, descriptions of each are below
         usePackageVersion,
         stripDebug
     },
+    logger: {
```

Comment:
Need to add a description of these new properties to the docs below.

Reply:
Fixed.